std/datetime/systime.d

```diff
@@ -2523,7 +2523,7 @@ public:
 
     version (StdDdoc)
     {
-        private struct timespec {}
+        version (Windows) private struct timespec {}
```

Comment:
This is to avoid
>/home/dtest/DAutoTest/work/tmp/ddoc_preprocessor_062038e10dce4e6eb0a29d181365ced6/std/file.d(1557): Error: cannot implicitly convert expression `accessTime.toTimeSpec()` of type `std.datetime.systime.SysTime.timespec` to `core.sys.posix.signal.timespec`